docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2943,105 +2943,106 @@ noise added to the data. We currently divide noise into two types:
 #. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
 #. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
 
-As there are multiple noise functions that will independently select 
-data. Noise should be added in the order below. Note that not all 
-noise functions apply to add data types, but this ordering should 
-be able to be used for all data. 
+Noise should be added in the order below.
 
 #. "Borrowed" SSN (happens in simulation NOT noise functions)
-#. Omissions 
-#. Duplications 
-#. Missing Data 
-#. Incorrect Selection 
-#. Copy from Within Household 
+#. Omissions
+#. Duplications
+#. Missing Data
+#. Incorrect Selection
+#. Copy from Within Household
 #. Month and day swaps (applies to dates only)
 #. Zip Code Miswriting (applies to Zip Code only)
 #. Age Miswriting (applies to age only)
-#. Numeric miswriting 
-#. Nicknames 
+#. Numeric miswriting
+#. Nicknames
 #. Fake names
-#. Phonetic 
-#. OCR 
-#. Typographic 
+#. Phonetic
+#. OCR
+#. Typographic
+
+.. note::
+
+  Not all noise functions apply to all observers and columns, but the subset of noise functions
+  applied should always be applied in the order they appear in this list.
 
 Column Noise
 ''''''''''''
 
 To begin, we will start with defining column based noise. Some general rules 
 for all column based noise include: 
 
-- Rows eligible for errors is the probability of selecting a row to have a particular type of noise added. For example, 1% noise level for incorrect selection to type of tax form means 1% of rows will be selected to have the wrong value selected. This is somewhat more complicated for: OCR, phonetic, typographic, and numeric miswriting which is elaborated on below. 
-- Note that selection for column noise is applied to NON-MISSING values only. This means if we select 5% for noise, that 5% is AFTER missing data has been removed, meaning it might be a smaller percent of the full sample. 
-- Token noise level is a noise parameter that only applies to certian noise types and defines the amount of errors expected once a string is selected for noise. This parameter is also elaborated on below. 
+- The row noise probability is the probability of selecting a row to have a particular type of noise added. For example, 0.01 row noise probability for incorrect selection on the "type of tax form" column means each row will have 0.01 (1%) probability to have the wrong value selected. In the case of OCR, phonetic, typographic, and numeric miswriting, it is not the case that every row selected will actually have any noise applied -- this is explained further below. 
```

Comment:
It just occurred to me that we're calling the probability of selecting a row the "row noise probability," but we're using "row noise" to mean something completely different in the Row Noise section. This could be confusing...

Reply:
Good point. I'll use "row-level noise probability" to disambiguate this.